Automattic-Tracks-iOS/UI/Crash Logging/CrashLoggingView.swift

```diff
@@ -0,0 +1,97 @@
+import SwiftUI
+import Sentry
+
+@available(iOS 13.0, OSX 10.15, *)
+public struct CrashLoggingView: View {
+
+    @State
+    var sendErrorAndWaitStatus: SendErrorAndWaitStatus = .none
+
+    @State
+    var sendingError: Error?
+
+    private let crashLogging: CrashLogging
+
+    public init(crashLogging: CrashLogging) {
+        self.crashLogging = crashLogging
+    }
+
+    public var body: some View {
+        Form {
+            Section(header: Text("Actions")) {
+                Button("Send Test Crash", action: sendTestCrash)
+                Button("Send Test Event", action: sendTestEvent)
+                HStack {
+                    Button(action: sendErrorAndWait) {
```

Comment:
I'm obviously over thinking this, given this is a debug view, but... as far as I can see, there is no logic to prevent running the action when the state is `.uploading`.

Reply:
Agreed. I had to strongly resist the urge to over-build this because it _is_ a demo app, so I'll stay strong and leave this for now.

Though it will bug me lol